README.md

```diff
@@ -1 +1,33 @@
-# example-backend-ruby-sinatra
+
+# Example Shunter Application Back End (Ruby, Sinatra)
+
+A Ruby and Sinatra-based back end for the [Shunter example application](https://github.com/shunterjs/example).
+
+
+## Requirements
+
+`Ruby 2.2.2+`
```

Comment:
If you really do require that version (versus you recommend it) then it might be an idea to require it in the Gemfile. If it's just a suggestion, you might want to add a `.ruby.version.example` file so rvm/rbenv users have an easy way to get started.

Neither is necessary, just being friendly to other devs.